samples/nrf9160/aws_fota/README.rst

```diff
@@ -29,6 +31,57 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+Prerequisites
+*************
+
+The connected device must have the same MQTT Client ID as the thing name created on AWS IoT.
+The device needs to have certificates from AWS IoT provisioned onto the device and it needs to have a policy attached to the certificates, and that the certificates are activate in AWS IoT. The process for doing so is described below.
+
+Below is an example of a very permissive policy which should be used for **testing only**.
+You can attach it to the certificates if you are unsure what policies are needed for your device.
+To create a policy, go to `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and select **Secure->Policies->Create**.
+
+.. code-block:: javascript
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": "iot:*",
+            "Resource": "*"
+          }
+       ]
+     }
+
+To create a thing, log in to the `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and select **Manage->Things->Create->Create a single thing**.
+Provide it with a name which corresponds to your device's MQTT Client ID.
+Next, choose to either to create certificates or add your own certificates and make sure to **Activate** them.
+Also make sure to download one of the `root CA certificates <https://docs.aws.amazon.com/iot/latest/developerguide/managing-device-certs.html#server-authentication>`_ and the generated certificates if you chose to let AWS generate some for layou.
```

Comment:
```suggestion
Also, make sure to download one of the `root CA certificates <https://docs.aws.amazon.com/iot/latest/developerguide/managing-device-certs.html#server-authentication>`_ and the generated certificates if you choose to let AWS generate some for you.
```

So this step should be done if you choose to create certificates in AWS console, rather than add your own certificates in the previous step?